src/components/pages/WinnerPage/RenderWinnerPage.js

```diff
@@ -4,33 +4,41 @@ import { Button } from 'antd';
 import { useHistory } from 'react-router-dom';
 import { connect } from 'react-redux';
 
+import explosion from '../../../assets/images/gamemodeimg/explosion.png';
+import boyImg from '../../../assets/images/gamemodeimg/LightingKid.png';
+const RenderWinnerPage = props => {
+  const { push } = useHistory();
+
+  const goToChildDashboard = () => {
```

Comment:
This page need to go to Play Again page.  Add button that redirects to child/play-again.